app/logic/util/util.ts

```diff
@@ -128,6 +128,12 @@ export class NotImplemented extends Error {
   }
 }
 
+export class NotSupported extends Error {
+  constructor(msg?: string) {
+    super(msg ?? "Operation not supported");
```

Comment:
User-visible strings must be translated. Same elsewhere.

```suggestion
    super(msg ?? gt`Operation not supported`);
```

Reply:
I was just copying `NotImplemented` above...